tests/test_core.py

```diff
@@ -65,6 +66,38 @@ def rao(freq_dirs):
     return rao
 
 
+@pytest.fixture
+def rao_for_mirroring_deg():
+
+    freq = np.linspace(0, 1.0, 3)
+    dirs = np.linspace(0, 180, 3, endpoint=True)
+    vals = np.array(
+        [
+            [5.0 + 9.0j, 1.0 - 4.0j, 3.0 - 2.0j],
+            [7.0 - 9.0j, 2.0 + 2.0j, 10.0 + 9.0j],
+            [6.0 + 1.0j, 1.0 - 4.0j, 7.0 + 3.0j],
+        ]
+    )
+
+    return wr.RAO(freq, dirs, vals, degrees=True)
+
+
+@pytest.fixture
+def rao_for_mirroring_rad():
+
+    freq = np.linspace(0, 1.0, 3)
+    dirs = np.linspace(0, np.pi, 3, endpoint=True)
+    vals = np.array(
+        [
+            [5.0 + 9.0j, 1.0 - 4.0j, 3.0 - 2.0j],
+            [7.0 - 9.0j, 2.0 + 2.0j, 10.0 + 9.0j],
+            [6.0 + 1.0j, 1.0 - 4.0j, 7.0 + 3.0j],
+        ]
+    )
+
+    return wr.RAO(freq, dirs, vals, degrees=False)
```

Comment:
Since this rao is only used by one test, I think it is better to just define it inside the test function instead of making a fixture

Reply:
But this is not important to fix. Do it if you agree